README.md

```diff
@@ -1,56 +1,104 @@
+
+[Changelog]: docs/Changelog.md
+[Schematic]: docs/Schematic.pdf
+[Preview]: docs/Preview.png
+[Vendors]: docs/Vendors.md
+[PCB]: cad
+
+[Green Pro Micro]: http://www.40percent.club/2017/09/green-pro-micro.html
+[License]: https://creativecommons.org/licenses/by-sa/4.0/
+
+[Alvaro]: https://github.com/Ariamelon/Alvaro
+[Comet]: https://github.com/vattern/comet
+[Elite-C]: https://keeb.io/products/elite-c-low-profile-version-usb-c-pro-micro-replacement-atmega32u4
+[Goldfish-clone-JLCPCB]: https://github.com/JeremyJStarcher/Goldfish-clone-JLCPCB
+
+<!----------------------------------------------------------------------------->
+
 # Goldfish
-The goldfish is a Pro Micro compatible microcontroller designed for use in mechanical keyboards, although it is perfectly suitable for any other hobby electronics project.
-It is based on the same ATMega32U4 chip as the original Pro Micro and can be programmed using the Arduino IDE, or keyboard specific firmware such as TMK or QMK.
+
+This **Pro Micro** compatible board designed for use in `Mechanical Keyboards`, <br>
+based on the same `ATMega32U4` chip and programmable using the **Arduino IDE** <br>
+or a keyboard specific firmware such as `TMK` or `QMK`.
+
+![Preview]
+
+*Of course it is perfectly usable for any other electronics projects as well.*
+
+---
+
+**⸢ [PCB] ⸥ ⸢ [Schematic] ⸥ ⸢ [Vendors] ⸥ ⸢ [Changelog] ⸥**
+
+---
 
 ## Advantages
-* The through-hole USB connector is much less likely to break off compared to surface mount connectors.
-* USB-C is reversible, more durable and just plain cool.
-* Pin-compatible with the original Pro Micro and Pro Micro clones.
-* Goldfish has five extra pins available along its bottom edge if more pins are required.
-* USB data lines are easily accessible through two internal pins.
-* A mid-mounted USB-C connector allows goldfish to be much thinner than a Pro Micro.
+
+- ***Lower board profile*** with the mid-mounted `USB-C` connector.
+
+- Two internal pins make the **USB** data lines easily accessible.
+
+- `USB-C` is more ***durable***, ***reversible*** and just *plain cool*.
+
+- `5` additional pins on the underside of the board.
+
+- Pin-compatible with the original **Pro Micro**.
+
+- ***Sturdier*** through-hole **USB** connector.
```

Comment:
Same thing I said earlier about excessive **bold** and `code` in sentences.

Also please use single spacing for the bullet points. It ends up being a bit more compact when rendered by GitHub, which I think looks better